packages/content-management/table_list_view_table/src/components/tag_filter_panel.tsx

```diff
@@ -176,8 +176,8 @@ export const TagFilterPanel: FC<Props> = ({
 
             <EuiFlexItem css={saveBtnWrapperCSS}>
               <EuiButton onClick={closePopover}>
-                {i18n.translate('contentManagement.tableList.tagFilterPanel.applyButtonLabel', {
-                  defaultMessage: 'Apply',
+                {i18n.translate('contentManagement.tableList.tagFilterPanel.doneButtonLabel', {
+                  defaultMessage: 'Done',
```

Comment:
It used to be `Save`, then we decided to call it `Apply`, then now `Done`.. naming is hard 😊

Reply:
And thanks for keeping the button, we need it for accessibility 👍